### PR TITLE
Add contiguous as pointwise rule

### PR DESCRIPTION
**Pull request by aazzolini**
Question, what's the best way to add a test for this op?

### CODE REVIEW

File: spmd/tensor/ops/pointwise_ops.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 
 linear_pointwise_ops = [
+    "aten.contiguous.default",
```
Comment (wanchaol) [inline spmd/tensor/ops/pointwise_ops.py:23]: Could we make this a tensor_ops instead? I feel we don't need to do resharding for contiguous(), it should just preserve the existing sharding?

**Review by wanchaol — COMMENTED** [review]
There're some existing tests in https://github.com/pytorch/tau/blob/815aa87e71961af87dcc85b1a85728275554e099/test/spmd/tensor/test_tensor_ops.py#L36 that test the contiguous(), maybe manually add some tests there?

File: spmd/tensor/ops/pointwise_ops.py (hunk 1 above)
Comment (aazzolini) [reply spmd/tensor/ops/pointwise_ops.py:23]: oh of course, here, right? https://github.com/pytorch/tau/blob/main/spmd/tensor/ops/tensor_ops.py#L80

Comment (wanchaol) [reply spmd/tensor/ops/pointwise_ops.py:23]: Yeah that should be the right place

**Review by wanchaol — APPROVED** [review]
Nice!

**Pull request merged**